packages/core/src/2d/assembler/SlicedSpriteAssembler.ts

```diff
@@ -14,7 +16,7 @@ export class SlicedSpriteAssembler {
   ];
   static _worldMatrix = new Matrix();
 
-  static resetData(renderer: SpriteRenderer): void {
+  static resetData(renderer: SpriteRenderer | UIImage): void {
```

Comment:
Same as the previous comment regarding SpriteRenderer | SpriteMask | UIImage. To avoid redundancy, consider using the same Renderable type here as well. See the comment in packages/core/src/2d/assembler/SimpleSpriteAssembler.ts